docs/lexical-search.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        api_key=api_key,
</pre><p>add text on removing source tag for prod/self</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/496/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>